pkgs/test/lib/dart.js

```diff
@@ -56,7 +56,7 @@ script.src = link.href + '.browser_test.dart.js';
 
 script.onerror = function(event) {
   var message = "Failed to load script at " + script.src +
-      (event.message ? ": " + event.message : ".");
+      (event.message ? ": " + event.message : ".") + " Make sure the script is being generated/processed.";
   sendLoadException(message);
```

Comment:
It might make more sense to handle this in the test runner, wherever we handle these load exceptions. The reason being we should be able to check if we are in "precompiled" mode at that point, and give a more explicit message about checking your build output (and mention build_web_compilers specifically).

Reply:
That's right, but in addition to the plenty more time that that would take me (never digged in build_runner code before) the error makes sense to be in the test package because it appears even without build_runner being called when running bad tests with -p [chrome|firefox] (or at least I think that happened to me while I was fighting the subject)

Reply:
I believe that message get's plumbed through here:
https://github.com/dart-lang/test/blob/dcd9b51808d762dae3fbc3a7d96180b7a87a3ba3/pkgs/test_core/lib/src/runner/plugin/platform_helpers.dart#L86